src/ert/enkf_main.py

```diff
@@ -148,6 +150,26 @@ def _seed_sequence(seed: Optional[int]) -> int:
     return int_seed
 
 
+def load_prior(
```

Comment:
Could we come up with a more descriptive name for this function? `save_design_matrix_to_ensemble` perhaps?

Reply:
I wanted to be consistent with the sample_priors (which also save the params to ensemble afterwards), but I think it is better to be descriptive.

Reply:
I've changed it as suggested. I've also replaced DesignMatrix into pd.DataFrame directly. I think I need to write a test for it.